docs/source/ops/connect/auth0.rst

```diff
@@ -58,15 +58,11 @@ In order to follow along this guide, you will need:
   requests from these IPs through.
 - To know the ``ledgerId`` your ledger self-identifies as. Refer to your
   specific driver's documentation for how to set the ``ledgerId`` value.
-- To be running SDK 1.17.0 or later. Before 1.17.0, the JSON API required an
-  extra token, the setup of which is not covered here. If you are somehow
-  unable to upgrade but still want to use Auth0, please contact us for
-  assistance.
 - An application you want to deploy on your Daml system. This is not, strictly
   speaking, required, but the whole experience is going to be a lot less
   satisfying if you don't end up with something actually running on your Daml
   system. In this guide, we'll use the `create-daml-app` template,
-  which as of Daml SDK 1.17.0 supports Auth0 out-of-the-box on its UI side.
+  which supports Auth0 out-of-the-box on its UI side.
```

Comment:
Feel free to change this, but we'll soon remove this page as it heavily depends on the Helm Chart (temporary measure until the Helm Chart is restored).

Reply:
The `create-daml-app` support for Auth0 predates, and does not depend on, the Helm chart. That said, I'm not claiming it still works in 2.0.